jobs/component_jobs.groovy

```diff
@@ -137,7 +137,8 @@ repos.each { Map repo ->
 
               export IMAGE_PREFIX=deisci VERSION="git-\${git_commit:0:7}"
               docker login -e="\$DOCKER_EMAIL" -u="\$DOCKER_USERNAME" -p="\$DOCKER_PASSWORD"
-              DEIS_REGISTRY='' make docker-build ${dockerPush}
+              # build once with "docker --pull --no-cache" to avoid stale layers
```

Comment:
perhaps we can just `export DOCKER_BUILD_FLAGS="--pull --no-cache"` above (like IMAGE_PREFIX and VERSION) so that both `docker-build` invocations use it (assuming this is desired) -- note the `quay.io` docker-build below...